spk/bazarr/src/service-setup.sh

```diff
@@ -15,14 +13,11 @@ SERVICE_COMMAND="$LANGUAGE $PYTHON ${SVC_CWD}/bazarr.py --no-update --config ${S
 
 service_postinst ()
 {
-
     # Create a Python virtualenv
-    ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env
-
-    wheelhouse=${SYNOPKG_PKGDEST}/share/wheelhouse
+    install_python_virtualenv
 
     # Install the wheels (using virtual env through PATH)
-    ${PIP} install --no-deps --no-index --upgrade --force-reinstall --find-links ${wheelhouse} ${wheelhouse}/*.whl
+    install_python_wheels
```

Comment:
It seems that this is not working as well as it used to as it seems to trigger the build for numpy and webrtcvad which obviously fail (since there's no compiler).

Could it be related to poor wheel namings ?

Reply:
@smaarn finally coming back to address the remaining issue found here.  Can you provide the logs and arch info?  As this should not occur since wheels should be available in the wheelhouse directory.

Reply:
webcrtvad issue still seems to be here

[bazarr.log](https://github.com/SynoCommunity/spksrc/files/7798929/bazarr.log)
.